cypress.config.ts

```diff
@@ -23,20 +24,28 @@ const getSpecPattern = (): string[] => {
 const baseUrl = (process.env.TEST_BASE_URL || 'https://localhost:8005').replace(/\/$/, '');
 
 export default defineConfig({
-  projectId:             process.env.TEST_PROJECT_ID,
-  defaultCommandTimeout: 60000,
-  trashAssetsBeforeRuns: true,
-  env:                   {
+  projectId:              process.env.TEST_PROJECT_ID,
+  defaultCommandTimeout:  60000,
+  trashAssetsBeforeRuns:  true,
+  env:                    {
     baseUrl,
-    username:          process.env.TEST_USERNAME,
-    password:          process.env.TEST_PASSWORD,
-    bootstrapPassword: process.env.CATTLE_BOOTSTRAP_PASSWORD,
+    coverage:             false,
+    codeCoverage: {
+      exclude: [
```

Comment:
It might be better to do an `includes` here on just `shell` and i guess `pkg`. Otherwise we should rule out anything in the root, stories, scripts and docusaurus in addition to these two

Reply:
That exclusion is to avoid Jest test files from the unit test to be evaluated as actual code.
We may add on top which one to include.
Anyhow, stories are `.mdx`, otherwise it may have sense to test them, while Docusaurus has only static configurations.

Reply:
Thanks for reminding me that we have still to exclude `/shell/scripts/` 👍